sql/catalyst/src/main/scala/org/apache/spark/sql/types/StructType.scala

```diff
@@ -519,6 +519,7 @@ case class StructType(fields: Array[StructField]) extends DataType with Seq[Stru
   private[sql] lazy val existenceDefaultValues: Array[Any] = getExistenceDefaultValues(this)
   private[sql] lazy val existenceDefaultsBitmask: Array[Boolean] = getExistenceDefaultsBitmask(this)
   private[sql] lazy val hasExistenceDefaultValues = existenceDefaultValues.exists(_ != null)
+  private[sql] lazy val defaultColumnAnalyzer: Analyzer = DefaultColumnAnalyzer
```

Comment:
Shall we remove the changes in this file?

Reply:
Sure, thanks for catching this, done.